include/s3select_functions.h

```diff
@@ -46,7 +46,9 @@ class s3select_functions : public __clt_allocator {
             {"extract", s3select_func_En_t::EXTRACT},
             {"dateadd", s3select_func_En_t::DATE_ADD},
             {"datediff", s3select_func_En_t::DATE_DIFF},
-            {"utcnow", s3select_func_En_t::UTCNOW}
+            {"utcnow", s3select_func_En_t::UTCNOW},
+            {"charlength", s3select_func_En_t::LENGTH},
+            {"characterlength", s3select_func_En_t::LENGTH}
```

Comment:
as we already  this issue 
the syntax parse should modified to accept underscore for function name
not critical for this PR , should note to ourselves, to fix it.